specs/system_info.table

```diff
@@ -0,0 +1,8 @@
+table_name("system_info")
```

Comment:
Also just noticed that the path of the spec file is `specs/system_info.table`. Since the current implementation is OS X specific, this file/path should be `specs/darwin/system_info.table` until we have a Linux equivalent otherwise it won't compile on Linux hosts.